docs/section4/4-6.md

```diff
@@ -0,0 +1,38 @@
+# Airside Systems – Rule 4-6
+
+**Rule ID:** 4-6
+**Rule Description:** HVAC fans shall remain on during unoccupied hours in systems primarily serving computer rooms.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 Schedules exception #3 for the proposed building  
+**Appendix G Section Reference:** None
+**Data Lookup:** None
+
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem 
+**Applicability Checks:** 
+
+1. Applies to hvac systems primarily serving computer rooms.
+
+**Function Call:** 
+
+1. get_zones_computer_rooms()
+
+**Manual Check:** None  
+ 
+**Rule Logic:**
+**Applicability Check 1:** 
+- Get list of zones that have computer rooms associated with them: `zones_serving_comp_rooms_list_p = get_zones_computer_rooms(P_RMR)`
+- For each zone in P_RMR: `for zone_p in P_RMR..Zone:`
+    - Check if the zone serves computer room(s), if yes then set zone serves computer room boolean variable to true: `if zone_p.id in zones_serving_comp_rooms_list_p: zone_primarily_serves_comp_room_check = TRUE`
```

Comment:
the terminology is slightly unclear. we say 'zone serves computer rooms' should this be 'zone_includes_computer_room'?

Reply:
Implemented with function. See function.